Panels/FrameTrackViewer.js

```diff
@@ -1235,16 +1237,20 @@ define([
             theLeftPanel._setFrame(theFrame);
             theRightPanel._setFrame(theFrame);
 
+            theFrame.includesToolBox = toolBoxWidth && toolBoxWidth > 0;
```

Comment:
It works  but it's confusing that includesToolBox will be an Number instead of an expected Boolean.
So I would suggest (you don't need that toolBoxWidth check anyway):
```suggestion
            theFrame.includesToolBox = toolBoxWidth > 0;
```

Reply:
True, good point, I commited the suggested change.